### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by kallewesterling**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+title: "Potential bug: <title>"
+labels: [
+  "bug"
+]
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: "Description"
+      description: Please enter an explicit description of your issue
```
Comment (tom-andersson) [inline .github/ISSUE_TEMPLATE/BUG-REPORT.yml:12]: 'explicit' has two meanings, perhaps we should say 'specific'?

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+      value: |
+        ![DESCRIPTION](LINK.png)
+      render: bash
+    validations:
+      required: false
+  - type: dropdown
+    id: os
+    attributes:
+      label: "OS"
+      description: What is the impacted OS environment?
```
Comment (tom-andersson) [inline .github/ISSUE_TEMPLATE/BUG-REPORT.yml:50]: Not sure about "impacted environment", how about "Which OS are you using?"

**Review by tom-andersson — CHANGES_REQUESTED** [review]
Looks great, thank you @kallewesterling, I just left a couple minor comments in the issue template.

**Comment by kallewesterling** [comment]
Edited and ready for review or merge, @tom-andersson !

**Comment by tom-andersson** [comment]
Amazing, thanks @kallewesterling!

**Pull request merged**